yarn.lock

```diff
@@ -7778,13 +7878,20 @@ webpack-sources@^1.0.1, webpack-sources@^1.1.0:
     source-list-map "^2.0.0"
     source-map "~0.6.1"
 
+webpack-sources@^1.2.0:
```

Comment:
Nice, this should improve the CI build as well. As people are claiming good improvements on webpack when having sourcemaps enabled.

Reply:
Would be cool to benchmark it the before and after with #12022. Only after the CI error is addressed though.